test/functional/p2p_invalid_messages.py

```diff
@@ -19,7 +22,8 @@
 from test_framework.test_framework import BitcoinTestFramework
 
 MSG_LIMIT = 4 * 1000 * 1000  # 4MB, per MAX_PROTOCOL_MESSAGE_LENGTH
```

Comment:
```suggestion
MSG_LIMIT = MAX_PROTOCOL_MESSAGE_LENGTH
```

Reply:
Good eye; that's an error introduced this morning in my last rebase, will fix

Reply:
That line should be removed.

Reply:
Removed the line.